docs/documentation/concepts/units-of-measurement.md

```diff
@@ -71,3 +71,116 @@ When creating QuantityType states the framework offers some useful packages and
+| Energy                 | Watt Second             | Ws     |
+| Energy                 | Watt Hour               | Wh     |
+| Energy                 | KiloWatt Hour           | kWh    |
+| Force                  | Newton                  | N      |
+| Frequency              | Hertz                   | Hz     |
+| Illuminance            | Lux                     | lx     |
+| Intensity              | Irradiance              | W/m²   |
+| Length                 | Metre                   | m      |
+| Length                 | Kilometre               | km     |
+| LuminousFlux           | Lumen                   | lm     |
+| LuminousIntensity      | Candela                 | cd     |
+| MagneticFlux           | Weber                   | Wb     |
+| MagneticFluxDensity    | Tesla                   | T      |
+| Mass                   | Kilogram                | kg     |
+| Mass                   | Gram                    | g      |
+| Power                  | Watt                    | W      |
+| Pressure               | Pascal                  | Pa     |
+| Pressure               | hectoPascal             | hPa    |
+| Pressure               | Millimetre of Mercury   | mmHg   |
+| Radioactivity          | Becquerel               | Bq     |
+| RadiationDoseAbsorbed  | Gray                    | Gy     |
+| RadiationDoseEffective | Sievert                 | Sv     |
+| SolidAngle             | Steradian               | sr     |
+| Speed                  | Metre per Second        | m/s    |
+| Speed                  | Kilometre per Hour      | km/h   |
+| Speed                  | Knot                    | kn     |
+| Temperature            | Kelvin                  | K      |
+| Temperature            | Celsius                 | °C     |
+| Time                   | Second                  | s      |
+| Time                   | Minute                  | min    |
+| Time                   | Hour                    | h      |
+| Time                   | Day                     | d      |
+| Time                   | Week                    | week   |
+| Time                   | Year                    | y      |
+| Volume                 | Cubic Metre             | m3     |
+
+Prefixes:
+
+| Name  | Symbol | Value |
+|-------|--------|-------|
+| Yotta | Y      | 10²⁴  |
+| Zetta | Z      | 10²¹  |
+| Exa   | E      | 10¹⁸  |
+| Peta  | P      | 10¹⁵  |
+| Tera  | T      | 10¹²  |
+| Giga  | G      | 10⁹   |
+| Mega  | M      | 10⁶   |
+| Kilo  | k      | 10³   |
+| Hecto | h      | 10²   |
+| Deca  | da     | 10    |
+| Deci  | d      | 10⁻¹  |
+| Centi | c      | 10⁻²  |
+| Milli | m      | 10⁻³  |
+| Micro | µ      | 10⁻⁶  |
+| Nano  | n      | 10⁻⁹  |
+| Pico  | p      | 10⁻¹² |
+| Femto | f      | 10⁻¹⁵ |
+| Atto  | a      | 10⁻¹⁸ |
+| Zepto | z      | 10⁻²¹ |
+| Yocto | y      | 10⁻²⁴ |
+
+
+To use the prefixes simply add the prefix to the unit symbol e.g.
+
+Examples:
+-milliAmpere - `mA`
+-centiMetre - `cm`
+kiloWatt - `kW`
```

Comment:
the two lines above start with a dash. Is it missing here?

Reply:
Yes it is for some odd oversight
I need to add a new unit in there
This will be tomorrow now
Thanks